exercises/practice/secret-handshake/.meta/example.nim

```diff
@@ -1,22 +1,18 @@
 import std/algorithm
 
-const
-  signals = [
-    "wink",
-    "double blink",
-    "close your eyes",
-    "jump"
-  ]
-  reverseBit = 4
+type
+  Action* = enum
+    Wink, DoubleBlink, CloseEyes, Jump, Reverse
 
+func has(n: Natural, action: Action): bool =
+  (n and 1 shl action.ord) != 0
```

Comment:
Can we use [`bitops.testBit`](https://nim-lang.org/docs/bitops.html#testBit%2CT%2CBitsRange%5BT%3A%20SomeInteger%5D)?
```suggestion
  testBit(n, action.ord)
```

Reply:
Ya. I do see your point. Either way is good for me